lib/console.api.js

```diff
@@ -26,48 +26,45 @@ class Console {
    * @returns {object} promise object.
    */
   log(message, type, errorCode, suppress) {
-    return new Promise((resolve, reject) => {
-      let entry = false;
-      let typeColors = {
-        status: chalk.dim,
-        notice: chalk.cyan,
-        warning: chalk.yellow.bold,
-        error: chalk.white.bgRed.bold,
-        success: chalk.green
-      };
+    let entry = false;
+    let typeColors = {
+      status: chalk.dim,
+      notice: chalk.cyan,
+      warning: chalk.yellow.bold,
+      error: chalk.white.bgRed.bold,
+      success: chalk.green
+    };
 
-      // If suppress is turned on, do nothing.
-      if (suppress) {
-        return;
-      }
+    // If suppress is turned on, do nothing.
+    if (suppress) {
+      return;
+    }
 
-      // Default type and error code.
-      type = type || 'status';
-      errorCode = errorCode || 1;
+    // Default type and error code.
+    type = type || 'status';
+    errorCode = errorCode || 1;
 
-      // If type is valid (recognizeable string), create colored message.
-      if (typeColors.hasOwnProperty(type)) {
-        entry = typeColors[type](message);
-      }
+    // If type is valid (recognizeable string), create colored message.
+    if (typeColors.hasOwnProperty(type)) {
+      entry = typeColors[type](message);
+    }
 
-      // If type is invalid, return message and let the user know.
-      else {
-        entry = message + ' (NOTE: "' + type + '" is not a proper log type. Please use "notice", "warning", "error", "status", or "success")';
-      }
+    // If type is invalid, return message and let the user know.
+    else {
+      entry = message + ' (NOTE: "' + type + '" is not a proper log type. Please use "notice", "warning", "error", "status", or "success")';
+    }
 
-      // If this is an error, log as an error.
-      if (type === 'error') {
-        console.error(message);
-        process.exit(errorCode);
-      }
+    // If this is an error, log as an error.
+    if (type === 'error') {
+      console.error(message);
+      process.exit(errorCode);
+    }
 
-      // If this is any other type of message, log as a simple string.
-      else {
-        console.log(entry);
-      }
-
-      resolve();
-    })
+    // If this is any other type of message, log as a simple string.
+    else {
+      console.log(entry);
+      return entry;
```

Comment:
Other than removing the promise related code this was the only other actual code change in this file.  The rest was adjusting whitespace.

We weren't returning the message before so the test had no string to test.  Let me know if you think there is another way to handle this.